crates/torii/core/src/sql/utils.rs

```diff
@@ -3,11 +3,19 @@ use starknet_crypto::Felt;
 
 use super::FELT_DELIMITER;
 
-pub fn felts_sql_string(felts: &[Felt]) -> String {
+pub fn felts_to_sql_string(felts: &[Felt]) -> String {
     felts.iter().map(|k| format!("{:#x}", k)).collect::<Vec<String>>().join(FELT_DELIMITER)
         + FELT_DELIMITER
 }
 
+pub(crate) fn felt_to_sql_string(felt: &Felt) -> String {
+    format!("{:#x}", felt)
```

Comment:
We should perhaps have a function with padding and one without padding? Or we always use without padding?

Reply:
yeah for felt we where not padding previously so i kept the same behaviour